From a8b00ad1ccdffd158db2cf5e8ff02cc1f7092d7d Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Tue, 23 Aug 2016 15:40:49 +0800
Subject: [PATCH] arm64: dts: rockchip: disable PCIe and PCIe-phy on rk3399-evb

Let's disable it as the auto link training of PCIe
take quite long time without add-in card or M.2
devices available.

Change-Id: I4a48a44574b68da75845a6e614a9970bb5d6685b
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
index 6c3cae823d34..c81c649ad086 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
@@ -381,15 +381,16 @@
 	};
 };
 
+&pcie_phy {
+	status = "disabled";
+};
+
 &pcie0 {
-	assigned-clocks = <&cru SCLK_PCIEPHY_REF>;
-	assigned-clock-parents = <&cru SCLK_PCIEPHY_REF100M>;
-	assigned-clock-rates = <100000000>;
 	ep-gpios = <&gpio3 13 GPIO_ACTIVE_HIGH>;
 	num-lanes = <4>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pcie_clkreqn>;
-	status = "okay";
+	status = "disabled";
 };
 
 &tcphy0 {
-- 
2.35.3

